$MirOS: ports/misc/mc/patches/patch-doc_mc_1_in,v 1.2 2006/02/20 20:23:22 tg Exp $
--- doc/mc.1.in.orig	Wed Jun  8 12:26:44 2005
+++ doc/mc.1.in	Mon Feb 20 18:32:46 2006
@@ -1,5 +1,5 @@
 .\"TOPICS "Topics:"
-.TH MC 1 "June 2005" "MC Version 4.6.1-pre4" "GNU Midnight Commander"
+.TH MC 1 "June 2005" "MC Version 4.6.1" "GNU Midnight Commander"
 .\"SKIP_SECTION"
 .SH "NAME"
 mc \- Visual shell for Unix-like systems.
@@ -186,7 +186,7 @@ Directory Tree\&.
 The default auto repeat rate for the mouse buttons is 400
 milliseconds. This may be changed to other values by editing the
 .\"LINK2"
-\&~/.mc/ini
+\&~/.etc/mc/ini
 .\"Save Setup"
 file and changing the
 .I mouse_repeat_rate
@@ -1216,7 +1216,7 @@ Directories to be skipped should be set 
 .B find_ignore_dirs
 in the
 .B Misc
-section of your ~/.mc/ini file.
+section of your ~/.etc/mc/ini file.
 .PP
 Directory components should be separated with a colon, here is an
 example:
@@ -1282,7 +1282,7 @@ description.
 .\"NODE "    Extension File Edit"
 .SH "    Extension File Edit"
 This will invoke your editor on the file
-.IR ~/.mc/bindings .
+.IR ~/.etc/mc/bindings .
 The format of this file following:
 .PP
 All lines starting with # or empty lines are thrown away.
@@ -1375,7 +1375,7 @@ The user menu is a menu of useful action
 the user. When you access the user menu, the
 file .mc.menu from the current directory is used if it exists,
 but only if it is owned by user or root and is not world-writable.
-If no such file found, ~/.mc/menu is tried in the same way,
+If no such file found, ~/.etc/mc/menu is tried in the same way,
 and otherwise mc uses the default system-wide menu
 @prefix@/share/mc/mc.menu.
 .PP
@@ -1658,7 +1658,7 @@ ones described in ed(1).
 .I Auto Save Setup.
 If this option is enabled, when you exit the Midnight Commander the
 configurable options of the Midnight Commander are saved in the
-~/.mc/ini file.
+~/.etc/mc/ini file.
 .PP
 .I Auto menus.
 If this option is enabled, the user menu will be invoked at startup.
@@ -1809,7 +1809,7 @@ and wait.
 .PP
 When you finish with all the keys, you can Save them.  The definitions
 for the keys you have redefined will be written into the [terminal:TERM]
-section of your ~/.mc/ini file (where TERM is the name of your current
+section of your ~/.etc/mc/ini file (where TERM is the name of your current
 terminal).  The definitions of the keys that were already working properly
 are not saved.
 .\"NODE "    Virtual FS"
@@ -1901,14 +1901,14 @@ firewalls.
 .\"NODE "    Save Setup"
 .SH "    Save Setup"
 At startup the Midnight Commander will try to load initialization
-information from the ~/.mc/ini file. If this file doesn't exist, it will
+information from the ~/.etc/mc/ini file. If this file doesn't exist, it will
 load the information from the system-wide configuration file, located in
 @prefix@/share/mc/mc.ini. If the system-wide configuration file doesn't
 exist, MC uses the default settings.
 .PP
 The
 .I Save Setup
-command creates the ~/.mc/ini file by saving the current settings
+command creates the ~/.etc/mc/ini file by saving the current settings
 of the
 .\"LINK2"
 Left, Right
@@ -2109,10 +2109,10 @@ valid until you quit the Midnight Comman
 If you are using
 .B bash
 you can specify startup
-commands for the subshell in your ~/.mc/bashrc file and
-special keyboard maps in the ~/.mc/inputrc file.
+commands for the subshell in your ~/.etc/mc/bashrc file and
+special keyboard maps in the ~/.etc/mc/inputrc file.
 .B tcsh
-users may specify startup commands in the ~/.mc/tcshrc file.
+users may specify startup commands in the ~/.etc/mc/tcshrc file.
 .PP
 When the subshell code is used, you can suspend applications at any
 time with the sequence C-o and jump back to the Midnight Commander, if
@@ -2517,7 +2517,7 @@ and then the assigned key. The macro is 
 you press Meta, Ctrl, or Esc and the assigned key, provided that the
 key is not used for any other function. Once defined, the macro
 commands go into the file
-.B .mc/cedit/cooledit.macros
+.B .etc/mc/cedit/cooledit.macros
 in your home directory. You can delete a macro by deleting the
 appropriate line in this file.
 .PP
@@ -2530,7 +2530,7 @@ code or another). This is controlled by 
 file
 .B @prefix@/share/mc/edit.indent.rc
 which is copied to
-.B .mc/cedit/edit.indent.rc
+.B .etc/mc/cedit/edit.indent.rc
 in your home directory the first time you use it.
 .PP
 You can use scanf search and replace to search and replace
@@ -3095,12 +3095,12 @@ Most of the settings of the Midnight Com
 menus. However, there are a small number of settings which can only be
 changed by editing the setup file.
 .PP
-These variables may be set in your ~/.mc/ini file:
+These variables may be set in your ~/.etc/mc/ini file:
 .TP
 .I clear_before_exec
 By default the Midnight Commander clears the screen before executing a
 command.  If you would prefer to see the output of the command at the
-bottom of the screen, edit your ~/.mc/ini file and change the value of
+bottom of the screen, edit your ~/.etc/mc/ini file and change the value of
 the field clear_before_exec to 0.
 .TP
 .I confirm_view_dir
@@ -3186,7 +3186,7 @@ contents of the selected directory.
 The Midnight Commander provides a way to fix your system terminal
 database without requiring root privileges.  The Midnight Commander
 searches in the system initialization file (the mc.lib file located in
-the Midnight Commander library directory) and in the ~/.mc/ini file for
+the Midnight Commander library directory) and in the ~/.etc/mc/ini file for
 the section "terminal:your-terminal-name" and then for the section
 "terminal:general", each line of the section contains a key symbol that
 you want to define, followed by an equal sign and the definition for the
@@ -3239,7 +3239,7 @@ The help file for the program.
 .IP
 The default system-wide extensions file.
 .PP
-.I ~/.mc/bindings
+.I ~/.etc/mc/bindings
 .IP
 User's own extension, view configuration and edit configuration
 file.  They override the contents of the system wide files if present.
@@ -3247,18 +3247,18 @@ file.  They override the contents of the
 .I @prefix@/share/mc/mc.ini
 .IP
 The default system-wide setup for the Midnight Commander, used only if
-the user doesn't have his own ~/.mc/ini file.
+the user doesn't have his own ~/.etc/mc/ini file.
 .PP
 .I @prefix@/share/mc/mc.lib
 .IP
 Global settings for the Midnight Commander.  Settings in this file
-affect all users, whether they have ~/.mc/ini or not.  Currently, only
+affect all users, whether they have ~/.etc/mc/ini or not.  Currently, only
 .\"LINK2"
 terminal settings
 .\"Terminal databases"
 are loaded from mc.lib.
 .PP
-.I ~/.mc/ini
+.I ~/.etc/mc/ini
 .IP
 User's own setup. If this file is present then the setup is loaded
 from here instead of the system-wide startup file.
@@ -3271,16 +3271,16 @@ This file contains the hints displayed b
 .IP
 This file contains the default system-wide applications menu.
 .PP
-.I ~/.mc/menu
+.I ~/.etc/mc/menu
 .IP
 User's own application menu. If this file is present it is used instead
 of the system-wide applications menu.
 .PP
-.I ~/.mc/Tree
+.I ~/.etc/mc/Tree
 .IP
 The directory list for the directory tree and tree view features.
 .PP
-.I ./.mc.menu
+.I ./.etc/mc.menu
 .IP
 Local user-defined menu. If this file is present, it is used instead of
 the home or system-wide applications menu.
